database/seeder/main.go

```
@@ -0,0 +1,191 @@
+    //
+    //// Create post views, comments, and likes
+    //for _, post := range allPosts {
+    //	createViews(db, post, 10)
+    //	createComments(db, post, 5, []database.User{admin, regular})
+    //	createLikes(db, post, []database.User{admin, regular})
+    //}
+    //
+    //fmt.Println("✅ Fake data seeding complete.")
+}
+
+func createUser(db *gorm.DB, name string, isAdmin bool) database.User {
```

golangci-lint (1.64.8)

func `createUser` is unused

(unused)